mantis/blackbox/main.py

```diff
@@ -15,26 +15,56 @@
 from simulation import data, optimal_routing
 import sys
 import os
-
+from typing import List
+from pydantic import BaseModel
+import pandas as pd
 
 app = FastAPI()
 
+@app.get("/xyk_pairs")
+async def xyk_pairs():
+    data = read_and_validate_csv()
+    return{"status": 200, "data": data}
```

Comment:
return just data, 200 is added automatically.

use AllData derived from stircty typed pandas(data frame) and pydantic model(json formatter for API).

and read transfers csv too.